libs/agno/agno/team/team.py

```diff
@@ -5450,9 +5487,20 @@ def transfer_task_to_member(
                     ):
                         yield ",".join([tool.get("content", "") for tool in member_agent_run_response_chunk.tools])
             else:
-                member_agent_run_response = member_agent.run(
-                    member_agent_task, images=images, videos=videos, audio=audio, files=files, stream=False
-                )
+                if not member_agent.knowledge_filters and member_agent.knowledge:
```

Comment:
Also for async? And there is a run function above as well? Maybe make it cleaner and build a `kwargs` dict that you can unpack for each call to `run`

Reply:
updated for async and streaming cases.